src/test/java/core/basesyntax/dao/RegistrationServiceTest.java

```diff
@@ -0,0 +1,118 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.RegistrationValidationException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @Test
+    void register_ValidUser_Ok() {
+        User user = new User();
+        user.setLogin("testlogin");
+        user.setPassword("password123");
+        user.setAge(25);
+
+        User registeredUser = registrationService.register(user);
+
+        assertNotNull(registeredUser);
+        assertEquals("testlogin", registeredUser.getLogin());
+        assertEquals("password123", registeredUser.getPassword());
+        assertEquals(25, registeredUser.getAge());
+        assertNotNull(registeredUser.getId());
+    }
+
+    @Test
+    void validLogin_Not_Ok() {

```

Comment:
The method name `validLogin_Not_Ok` does not follow the convention `<methodUnderTest>_<state>_<expectedBehavior>`. Consider renaming it to `register_loginShorterThanRequired_notOk` to reflect the specific case you are testing and the expected outcome.